### ISSUE TITLE
Forbid collection creation with unpacking without more arguments

### ISSUE THREAD

**Issue by orsinium** [wemake-services/wemake-python-styleguide#1101]
# Rule request

## Thesis

```python
t = (1, 2, 3)
# bad
[*t]
# good
list(t)

# ok
[4, *t, 5]
```

Same for `set` and `tuple`.

## Reasoning

Consistency :eyes:

**Comment by sobolevn** [comment]
Great idea!

**Comment by skarzi** [comment]
Can I take it?

**Comment by sobolevn** [comment]
Yes, sure! Let's schedule it after #1121 👍
